app/assets/javascripts/discourse/tests/integration/component-templates-test.gjs

```diff
@@ -16,6 +16,7 @@ function silenceMobileAndOverrideDeprecations(hooks) {
     const promise = new Promise((resolve) => (unsilenceCallback = resolve));
     withSilencedDeprecationsAsync(
       [
+        "component-template-resolving", // silence ember's template resolving deprecation
```

Comment:
Should we silence it in production as well?

Reply:
I was in doubt about that. AFAIK, it's only a warning, and it's not triggered in core anymore.

Do we want to silence it or have it enabled to collect the data?

Reply:
Ok cool, fine to leave as-is :+1: